lib/core/src/lib/form/components/widgets/core/form-field.model.ts

```diff
@@ -372,7 +378,7 @@ export class FormFieldModel extends FormWidgetModel {
 
                 if (typeof this.value === 'string') {
                     if (this.value === 'empty' || this.value === '') {
-                        delete this.form.values[this.id];
+                        this.form.values[this.id] = null;
```

Comment:
can we use undefined as in the other places ? 
https://github.com/Alfresco/alfresco-ng2-components/pull/8021/files#diff-ae592ea8f0e66bf49f5dcd46b7d6ad926e83b70fb3aed81c146cad30a15fc9a7R150

Reply:
The API expects null in order to unset the value of a dropdown field. undefined wouldn't work in this case